.env.example

```diff
@@ -1,4 +0,0 @@
-# See the `bumpversion.sh` for more information on these variables
-export TEAMS_BUMP_WEBHOOK="https://example.com/webhookb2/..." 
-export TWINE_USERNAME=""
-export TWINE_PASSWORD=""
```

Comment:
I'd prefer we hang on to this, I think I reference it in the docs for `bump_version.sh`. We should give it a more descriptive name though for sure, maybe just `.env.bumpversion.example`?

Reply:
Oh, sorry, I meant to delete just the kiln env example. I'll restore that.